app/globals.css

```
@@ -1,8 +1,25 @@
+    border-color: var(--color-gray-200, currentColor);
+  }
+}
 
 body {
   color: #2d1800;
   background: #2d1800
 }
+
+@plugin "daisyui";
+@plugin "daisyui/theme" {
+  name: "mikn";
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)